playbook.yml

```diff
@@ -1,77 +1,114 @@
+        dest: "{{ HPC_ENV_PREFIX }}/data/1000G/phase1/human_g1k_v37_phiX.fasta.ann"
+        checksum: 'md5:e858be5e40ee30ba8c0f584395188c0e'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/referenceGenome/human_g1k_v37_phiX.fasta.bwt"
+        dest: "{{ HPC_ENV_PREFIX }}/data/1000G/phase1/human_g1k_v37_phiX.fasta.bwt"
+        checksum: 'md5:c0570831ca6fff13ee6157eeb1813b88'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/referenceGenome/human_g1k_v37_phiX.fasta.fai"
+        dest: "{{ HPC_ENV_PREFIX }}/data/1000G/phase1/human_g1k_v37_phiX.fasta.fai"
+        checksum: 'md5:2bc786e8a1f6b71ba75c3368c2e50b19'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/referenceGenome/human_g1k_v37_phiX.fasta.pac"
+        dest: "{{ HPC_ENV_PREFIX }}/data/1000G/phase1/human_g1k_v37_phiX.fasta.pac"
+        checksum: 'md5:b42d311e535804ca80aa87f72a52c083'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/referenceGenome/human_g1k_v37_phiX.fasta.sa"
+        dest: "{{ HPC_ENV_PREFIX }}/data/1000G/phase1/human_g1k_v37_phiX.fasta.sa"
+        checksum: 'md5:20e20f4d6f884f5bf81d8beafe2eb098'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/snpEff/hg19.tar.gz"
+        dest: "{{ HPC_ENV_PREFIX }}/data/snpEff-4.3/hg19.tar.gz"
+        checksum: 'md5:75db3b74aac7eaa3295d4278443dbb70'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/inSilico.tar.gz"
+        dest: "{{ HPC_ENV_PREFIX }}/data/inSilico/inSilico.tar.gz"
+        checksum: 'md5:1b7e33b2977786819a2099e3895f623a'
+      - url: "https://molgenis26.gcc.rug.nl/downloads/NGS_DNA-resources/Agilent.tar.gz"
+        dest: "{{ HPC_ENV_PREFIX }}/data/Agilent/Agilent.tar.gz"
+        checksum: 'md5:0a76a2dde10ad1899cccc4f32f6ee804'
+      - url: "https://github.com/molgenis/easybuild-easyconfigs/releases/download/{{ pb_ebconfigs_version }}/easybuild-easyconfigs-{{ pb_ebconfigs_version }}-custom.tar.gz"
+        dest: "{{ easybuild_sources_dir }}/e/easyconfigs/{{ pb_ebconfigs_version }}-custom.tar.gz"
+        checksum: 'md5:259b8b100ceecdc434a2ecb2516556f4'
+  - name: Extract downloaded sources and reference data.
+    unarchive: src="{{ item.src }}" dest="{{ item.dest }}" remote_src=yes mode="{{ MODE_DIRS }}"
+    with_items:
+      - src:  "{{ HPC_ENV_PREFIX }}/data/snpEff-4.3/hg19.tar.gz"
+        dest: "{{ HPC_ENV_PREFIX }}/data/snpEff-4.3/"
+      - src:  "{{ HPC_ENV_PREFIX }}/data/inSilico/inSilico.tar.gz"
+        dest: "{{ HPC_ENV_PREFIX }}/data/"
+      - src:  "{{ HPC_ENV_PREFIX }}/data/Agilent/Agilent.tar.gz"
+        dest: "{{ HPC_ENV_PREFIX }}/data/Agilent/"
+      - src:  "{{ easybuild_sources_dir }}/e/easyconfigs/{{ pb_ebconfigs_version }}-custom.tar.gz"
+        dest: "{{ easybuild_software_dir }}/easyconfigs"
+  - name: Copy sources for software packages, which cannot be downloaded automagically by EB for technical or legal reasons.
```

Comment:
Are these software packages already in the cluster? Because according to this playbook they are not downloaded from a specific place